app/views/metadata/upload.html.slim

```diff
@@ -0,0 +1,12 @@
+.litebox-embed(style="width:550px")
+  h1 Upload Metadata
+  p To update metadata for several works within this collection, upload a spreadsheet with one row per work. See #{link_to "link", collection_metadata_example_path(params[:id])} for an example. (Note that multi-line/description fields cannot be updated.
+
+  =form_for(:metadata, url: collection_metadata_create_path) do |f|
+    table.form
+      tr.big
+        th =f.label :file, 'Metadata File'
+        td.w100.aright =f.file_field :file
```

Comment:
Please make this UI match the file upload UI for uploading works.

Reply:
@benwbrum done, thanks.